seat/src/contract.rs

```diff
@@ -565,9 +486,9 @@ mod tests {
         .unwrap();
         let active_primary_sale: QueryResp = from_binary(&active_primary_sale_query).unwrap();
 
-        match active_primary_sale {
-            QueryResp::ActivePrimarySale(Some(sale)) => assert!(sale.disabled),
-            _ => unreachable!(),
+        // there should be no active primary sale after sale is halted
+        if let QueryResp::ActivePrimarySale(Some(_sale)) = active_primary_sale {
+            panic!()
```

Comment:
In general, in tests, we want to say why things panicked, or let the panic result from an assertion that wasn't met